homeworks/igor.doroshenko_igordordev/3-async/style.css

```
@@ -0,0 +1,112 @@
+.header-panel__title::first-letter, .article__title::first-letter, .article__content::first-letter {
```

Став переноси рядочків після коми
```suggestion
.header-panel__title::first-letter,
.article__title::first-letter,
.article__content::first-letter {
```